### ISSUE TITLE
Batch delete

### ISSUE THREAD

**Issue by marin117** [marin117/Gmail-deleter#6]
How to speed up message deletion with [batch delete](https://developers.google.com/gmail/api/v1/reference/users/messages/batchDelete)

**Comment by marin117** [comment]
For some reason batch delete operation returns either *Backend error* or *Bad request*
Operation is implemented in [batch_delete](https://github.com/marin117/Gmail-deleter/tree/batch_delete) branch.

**Comment by adilhussain540** [comment]
@ marin117 - I saw the issues that you have created and thought I could contribute but it looks like the project is longer being maintained. I have recently used your project with some modifications for deleting more than half a million emails using `batchDelete` as the current implementation was way too slow.

Regarding the `batchDelete` errors, there's a limit on how many `IDs` one can send in a single request. I've tried multiple values and found out that the limit is 1000.
